feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/SessionTag.kt

```diff
@@ -0,0 +1,66 @@
+package io.github.droidkaigi.confsched2023.sessions.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import io.github.droidkaigi.confsched2023.designsystem.preview.MultiThemePreviews
+import io.github.droidkaigi.confsched2023.designsystem.theme.KaigiTheme
+import io.github.droidkaigi.confsched2023.designsystem.theme.hallColors
+
+@Composable
+fun SessionTag(
+    label: String,
+    modifier: Modifier = Modifier,
+    labelColor: Color = MaterialTheme.colorScheme.onSurfaceVariant,
+    backgroundColor: Color = Color.Transparent,
+    borderColor: Color = MaterialTheme.colorScheme.outline,
```

Comment:
[IMO]
It might be better to change its type to nullable with default null value, and add border modifier conditionally.

Reply:
Agreed. Fixed it!